cmd/benchmark/main.go

```diff
@@ -65,7 +67,7 @@ func parseInputParams() {
 	pflag.StringVarP(&output, "output", "o", defaultStdoutPath, "output file (use - for STDOUT)")
 	pflag.IntVarP(&concurrent, "concurrent", "c", 1, "concurrent users")
 	pflag.IntVarP(&repetitions, "repetitions", "r", 1, "repetitions for one user")
-	pflag.StringVarP(&memberKeys, "memberkeys", "k", "", "path to file with member keys")
+	pflag.StringVarP(&memberKeys, "memberkeys", "k", "", "path to dir with members keys")
```

Comment:
if you change the way how to use command flag you should fix all benchmark mentions, for example in `./docs` directory (grep all project files)